skopt/tests/test_space.py

```diff
@@ -375,6 +377,22 @@ def test_categorical_identity():
     assert_array_equal(transformed, samples)
     assert_array_equal(samples, cat.inverse_transform(transformed))
 
+@pytest.mark.fast_test
+def test_categorical_repr(capsys):
+
+    # ensure printing of Categorical objects does not
+    # raise an exception. Hhowever we do not insist
+    # on the exact output format.
+
+    for categories in (
+        range(3), # short ( < 7 items) list of categories
+        range(8), # long ( >= 7 items) list of categories with ellipsis
+        ):
```

Comment:
You can use something along the lines of `@pytest.mark.parametrize("categories", [range(5), range(9)])` to get rid of the for loop inside the test. I think there is an example of this decorator further down in this file.

Reply:
Thanks for pointing this out. Changed now in 9804c0e .

I checked with `pytest -s` that it actually executes the test twice with the correct parameters.